ts/Stock/Navigator/Navigator.ts

```diff
@@ -486,21 +491,54 @@ class Navigator {
         }
 
         // Create the handlers:
-        if (navigatorOptions.handles && navigatorOptions.handles.enabled) {
+        if (navigatorOptions.handles?.enabled) {
             const handlesOptions =
                 navigatorOptions.handles as Required<NavigatorHandlesOptions>,
                 { height, width } = handlesOptions;
 
             [0, 1].forEach((index: number): void => {
-                navigator.handles[index] = renderer.symbol(
-                    handlesOptions.symbols[index],
-                    -width / 2 - 1,
-                    0,
-                    width,
-                    height,
+                merge(
+                    true,
+                    navigator.handles[index] as NavigatorHandlesOptions,
```

Comment:
This merge seems to be wrong: do you merge `Renderer.symbol()` with `handlesOptions`? 

Edit: I'm not saying it's completely wrong, it is possible, however at least a comment why it's necessary is needed.

Reply:
I agree, after analyzing it, it is incorrect.